google/cloud/aiplatform/featurestore/featurestore.py

```diff
@@ -715,59 +749,14 @@ def _validate_and_get_batch_read_feature_values_request(
 
         return batch_read_feature_values_request
 
-    def _get_read_instances(
-        self, read_instances: Union[str, List[str]],
-    ) -> Union[gca_io.BigQuerySource, gca_io.CsvSource]:
-        """Gets read_instances
-
-        Args:
-            read_instances (Union[str, List[str]]):
-                Required. Read_instances can be either BigQuery URI to the input table,
-                or Google Cloud Storage URI(-s) to the csv file(s).
-
-        Returns:
-            Union[gca_io.BigQuerySource, gca_io.CsvSource]: BigQuery source or Csv source for read instances.
-
-        Raises:
-            TypeError if read_instances is not a string or a list of strings.
-            ValueError if read_instances uri does not start with 'bq://' or 'gs://'.
-            ValueError if uris in read_instances do not start with 'gs://'.
-        """
-        if isinstance(read_instances, str):
-            if not (
-                read_instances.startswith("bq://") or read_instances.startswith("gs://")
-            ):
-                raise ValueError(
-                    "The read_instances accepts a single uri starts with 'bq://' or 'gs://'."
-                )
-        elif isinstance(read_instances, list) and all(
-            [isinstance(e, str) for e in read_instances]
-        ):
-            if not all([e.startswith("gs://") for e in read_instances]):
-                raise ValueError(
-                    "The read_instances accepts a list of uris start with 'gs://' only."
-                )
-        else:
-            raise TypeError(
-                "The read_instances type should to be either a str or a List[str]."
-            )
-
-        if isinstance(read_instances, str):
-            if read_instances.startswith("bq://"):
-                return gca_io.BigQuerySource(input_uri=read_instances)
-            else:
-                read_instances = [read_instances]
-
-        return gca_io.CsvSource(gcs_source=gca_io.GcsSource(uris=read_instances))
-
     @base.optional_sync(return_input_arg="self")
     def batch_serve_to_bq(
         self,
         bq_destination_output_uri: str,
         serving_feature_ids: Dict[str, List[str]],
-        feature_destination_fields: Optional[Dict[str, str]] = None,
-        read_instances: Optional[Union[str, List[str]]] = None,
+        read_instances_uri: str,
```

Comment:
It looks like this is the case but just want to confirm the method hasn't been released yet. Thanks!

Reply:
Yes, it has not been released.